frontend/app/src/components/home/CurrentChatMessages.svelte

```diff
@@ -497,7 +498,7 @@
         return firstKey;
     }
 
-    $: groupedEvents = client.groupEvents(events, user.userId, groupInner(filteredProposals)).reverse();
+    $: groupedEvents = client.groupEvents(events, user.userId, $hideDeletedStore, groupInner(filteredProposals)).reverse();
```

Comment:
if the `hideDeletedStore` lives inside the client, we never need to pass its value to a method on the client.

Reply:
As discussed we need the components to "react" to the hideDeleted store changing